### PR TITLE
Unique Number in C

### PR DESCRIPTION
**Pull request by salonishah01**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Code for Unique Number - III in C

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#988

### CODE REVIEW

**Review by iamrajiv — APPROVED** [review]

File: C/other/UniqueNumberIII.c [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+2
+2
+6
+6 
+Output: 5
+
+Example 2:
+Sample Input: 
+Enter number of elements: 4
+Enter 7 elements: 3
```
Comment (atarax665) [inline C/other/UniqueNumberIII.c]:
```suggestion
Enter 4 elements: 3 

```

File: C/other/UniqueNumberIII.c [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    for(i=0;i<n;i++)
+    {
+        x = arr[i];
+        two |=one & x;
+        one ^=x;
+        n_t =~(one & two);
+        one &= n_t;
+        two &= n_t;
+    }
+    printf("The number that occurs only once: %d \n", one);
```
Comment (atarax665) [inline C/other/UniqueNumberIII.c]:
```suggestion
    printf("\nThe number that occurs only once is: %d \n", one);

```

File: C/other/UniqueNumberIII.c [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Output: 5
+
+Example 2:
+Sample Input: 
+Enter number of elements: 4
+Enter 7 elements: 3 
+3
+1
+3 
+Output: 1
```
Comment (atarax665) [inline C/other/UniqueNumberIII.c]: Sample output should be the output returned when the program is run in an IDE.
Something like ```The number that occurs only once is: 1```

**Review by atarax665 — COMMENTED** [review]
Format for code according to [C style guide](https://github.com/TesseractCoding/NeoAlgo/blob/master/C/C_Style_guide.md) and make the suggested changes.

**Comment by salonishah01** [comment]
@atarax665 Requested changes are done.

**Pull request merged**